### DISCUSSION TITLE
node-red and get operation

### DISCUSSION THREAD

**Discussion by alexblav** [vinodsr/node-red-contrib-tuya-smart-device#112] [Q&A] [answered]
Hi there,

First of all, thanks for the great work.
I'm new to node-red. I saw in the documentation that it is possible to get information from a device using GET.
Unfortunately, I don't see how to implement this in node-red. I have tried to write a function without success.
Could you please show me an example of syntax like you did with SET which allowed me to manage my shutters.

Thank you in advance.

**Comment by vinodsr** [comment]
For GET you need to pass the msg.payload as JSON
`{"operation":"GET" , "dps": 1}`

For getting the whole schema (if device supports) set msg.payload as 
`{"operation":"GET","schema":true}`

Hope this helps. 

I will try to add a example soon in the repo.

**Reply by alexblav** [reply, marked as answer]
thank you very much it works great
